enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/presencecheck/iproov/service/IProovRestApiService.java

```diff
@@ -162,45 +164,30 @@ public ResponseEntity<String> generateVerificationToken(OwnerId id) throws RestC
      * @return Response entity with the result json
      */
     public ResponseEntity<String> validateVerification(OwnerId id, String token) throws RestClientException {
-        ClaimValidateRequest request = new ClaimValidateRequest();
+        final ClaimValidateRequest request = new ClaimValidateRequest();
         request.setApiKey(configProps.getApiKey());
         request.setSecret(configProps.getApiSecret());
         request.setClient("Wultra Enrollment Server, activationId: " + id.getActivationId()); // TODO value from the device
         request.setIp("192.168.1.1"); // TODO deprecated but still required
         request.setRiskProfile(configProps.getRiskProfile());
         request.setToken(token);
 
-        String userId = getUserId(id);
+        final String userId = getUserId(id);
         request.setUserId(userId);
 
+        logger.info("Calling /claim/verify/validate userId={}, token={}, {}", userId, token, id);
```

Comment:
It seems insecure, but recommended by iProov, see https://docs.iproov.com/docs/Content/ImplementationGuide/api/api-logging.htm